public/includes/header.php

```diff
@@ -31,21 +31,24 @@
         <div class="collapse navbar-collapse" id="navbarSupportedContent">
           <ul class="navbar-nav mr-auto">
             <li class="nav-item">
-              <a class="nav-link" href="az.php">A-Z of BALL</span></a>
+              <a class="nav-link <?php if($currentPage =='index'){echo 'active';}?>" href="index.php">Home</span></a>
```

Comment:
Even though the code here is direct and straight forward, it is best practices to always abstract repeated code as a function. It makes it easier to update in case we want to update them later.